### ISSUE TITLE
Can't reproduce your results

### ISSUE THREAD

**Issue by lihui1998** [redrock303/NeRFLiX_CPVR2023#15]
Your work is great and I am very interested in your work. But if I don’t change the learning rate and train from scratch, NAN values will appear. Even if I change the learning rate, I can’t achieve the effect of the model you provided. I also found that the size of the model I trained was inconsistent compared to the model you provided. Why is this? Do you have any subsequent code modifications that are not synchronized? Hope you can answer, thank you very much.

**Comment by ZestfulJX** [comment]
> Your work is great and I am very interested in your work. But if I don’t change the learning rate and train from scratch, NAN values will appear. Even if I change the learning rate, I can’t achieve the effect of the model you provided. I also found that the size of the model I trained was inconsistent compared to the model you provided. Why is this? Do you have any subsequent code modifications that are not synchronized? Hope you can answer, thank you very much.

@lihui1998  Hello, I also encountered NaN during training, and the model would suddenly crash during the training process. I tried limiting the loss and gradient clipping methods, but neither solved the problem. Moreover, NaN occurrences would be accompanied by more memory consumption. I wonder if you have solved this problem. If you have any insights, I look forward to discussing them with you.
